lib/vagrant-openshift/resources/configure_docker.sh

```diff
@@ -23,24 +23,26 @@ elif lvdisplay centos >/dev/null 2>&1; then
 fi
 
 if [[ -n "${VG}" ]]; then
-    lvcreate -n docker-data -l 70%FREE /dev/${VG}
-    lvcreate -n docker-metadata -l 17%FREE /dev/${VG}
-    lvcreate -n openshift-xfs-vol-dir -l 100%FREE /dev/${VG}
+    lvcreate -n openshift-xfs-vol-dir -l 30%FREE /dev/${VG}
     mkfs.xfs /dev/${VG}/openshift-xfs-vol-dir
     mkdir -p /mnt/openshift-xfs-vol-dir
     echo /dev/${VG}/openshift-xfs-vol-dir /mnt/openshift-xfs-vol-dir xfs gquota 1 1 >> /etc/fstab
     mount /mnt/openshift-xfs-vol-dir
     chown -R "${SSH_USER}:${SSH_USER}" /mnt/openshift-xfs-vol-dir
 
-    DOCKER_STORAGE_OPTIONS="  -s devicemapper"
-    DOCKER_STORAGE_OPTIONS+=" --storage-opt dm.datadev=/dev/${VG}/docker-data"
-    DOCKER_STORAGE_OPTIONS+=" --storage-opt dm.metadatadev=/dev/${VG}/docker-metadata"
+    DOCKER_STORAGE_OPTIONS="-s devicemapper"
```

Comment:
Depends on what your goal is. If you want to use the out of the box config from docker-storage-setup so no one comes back to you and says the bug you filed is invalid because you're not using the supported docker storage configuration then I think you should stick with d-s-s defaults. If those defaults are wrong we should fix them.

If you truly have additional options that you think are optimal for your environment but not the defaults set them via `EXTRA_DOCKER_STORAGE_OPTIONS` 

FWIW, d-s-s sets `dm.use_deferred_removal=true` is for all versions of docker > 1.7 and `dm.use_deferred_deletion=true` for > 1.9 when the kernel supports it.